packages/dds/matrix/src/test/matrix.fuzz.spec.ts

```diff
@@ -103,31 +120,63 @@ const reducer = combineReducers<Operation, State>({
 interface GeneratorOptions {
 	insertRowWeight: number;
 	insertColWeight: number;
+	removeRowWeight: number;
+	removeColWeight: number;
 	setWeight: number;
 }
 
 const defaultOptions: GeneratorOptions = {
 	insertRowWeight: 1,
 	insertColWeight: 1,
+	removeRowWeight: 1,
+	removeColWeight: 1,
 	setWeight: 20,
 };
 
 function makeGenerator(optionsParam?: Partial<GeneratorOptions>): AsyncGenerator<Operation, State> {
-	const { setWeight, insertColWeight, insertRowWeight } = {
+	const { setWeight, insertColWeight, insertRowWeight, removeRowWeight, removeColWeight } = {
 		...defaultOptions,
 		...optionsParam,
 	};
 
+	const maxDimensionSizeChange = 1;
+
 	const insertRows: Generator<InsertRows, State> = ({ random, client }) => ({
 		type: "insertRows",
 		start: random.integer(0, client.channel.rowCount),
-		count: random.integer(1, 3),
+		count: random.integer(1, maxDimensionSizeChange),
```

Comment:
A bit curious about why we restrict the maximum change count for row/column insertion to just 1. If the limit is consistently set to 1, would it still be necessary to incorporate `random` here?

Reply:
thanks for catching, this was a temp change while I was investigating perf. I'll put this back to 3 (still gonna leave it refactored into a variable)